install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -173,8 +173,6 @@ spec:
                     echo "Node de-initialization complete"
 {{- end }}
           env:
-          - name: CHECKPOINT_PATH
-            value: /tmp/node-init.cilium.io
```

Comment:
Can we just check if the dynamic CHECKPOINT_PATH setting is shipped with 1.10?

https://github.com/cilium/image-tools/blob/396d5a19690229b42398a8be63b5712ea64a2601/images/startup-script/manage-startup-script.sh#L58

I'm not sure where this is included.

Reply:
It seems it was added 2 years ago so we should be safe I guess 

https://github.com/cilium/image-tools/commit/d2957c241002f7d46682c6e5308782f229636798

Reply:
I've checked, the image `quay.io/cilium/startup-script:62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8` contains the `CHECKPOINT_PATH` logic.